assets/js/src/core/components/tree/node/tree-node.tsx

```diff
@@ -153,8 +155,30 @@ const TreeNode = ({
     nodesRefs!.current[internalKey] = nodeRef
   }
 
+  function onDragOver (event: MouseEvent): void {
+    const assetMetaData = metaData?.asset
+
+    if (assetMetaData !== undefined && assetMetaData.type === 'folder') {
+      setSelectedIds([id])
+    }
+  }
+
+  const uploadProps: UploadProps = {
+    action: `/studio/api/assets/add/${id}`,
+    name: 'file',
+    multiple: true,
+    openFileDialogOnClick: false,
+    showUploadList: false,
+    onChange: ({ fileList }) => {
+      setUploadFileList(fileList.filter((file) => file.status === 'uploading'))
+    }
+  }
```

Comment:
Also not UI relevant should be moved to the asset tree under modules.

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/527/commits/7ac29efdc39882e0150aa4f8cc710aaa6b66fc4c